pkg/etcdcli/health_test.go

```diff
@@ -260,6 +260,67 @@ func healthyMember(member int) healthCheck {
 	}
 }
 
+func TestMinimumTolerableQuorum(t *testing.T) {
+
+	scenarios := []struct {
+		name   string
+		input  int
+		expErr error
+		exp    int
+	}{
+		{
+			name:   "valid input `3`",
+			input:  3,
+			expErr: nil,
+			exp:    2,
+		},
+		{
+			name:   "valid input `5`",
+			input:  5,
+			expErr: nil,
+			exp:    3,
+		},
+		{
+			name:   "invalid input `0`",
+			input:  0,
+			expErr: fmt.Errorf("invalid etcd member length: %v", 0),
+			exp:    0,
+		},
+		{
+			name:   "invalid input `-10`",
+			input:  -10,
+			expErr: fmt.Errorf("invalid etcd member length: %v", -10),
+			exp:    0,
+		},
+	}
+
+	for _, scenario := range scenarios {
+		t.Run(scenario.name, func(t *testing.T) {
+			actual, err := MinimumTolerableQuorum(scenario.input)
+
+			if scenario.expErr != nil {
```

Comment:
you can simplify all your if conditions below with a simple:

```
require.Equal(t, scenario.expErr, err)
require.Equal(t, scenario.exp, actual)
```

Reply:
thanks I never used this :D

Reply:
this is `testify` ?

Reply:
yes sir!